Lib/ctypes/test/test_anon.py

```diff
@@ -35,6 +36,18 @@ def test_anon_nonmember(self):
                                                       {"_fields_": [],
                                                        "_anonymous_": ["x"]}))
 
+    @test.support.cpython_only
+    def test_issue31490(self):
+        # There shouldn't be an assertion failure in case the class has an
+        # attribute whose name is specified in _anonymous_ but not in _fields_.
+
+        # AttributeError: 'x' is specified in _anonymous_ but not in _fields_
+        with self.assertRaises(AttributeError):
+            class Name(Structure):
+                _fields_ = []
```

Comment:
From the documentation:
> `_anonymous_` must be already defined when `_fields_` is assigned, otherwise it will have no effect.

Reply:
all right. should we change the other tests too?

Reply:
Hmm, if they are wrong, it should be a separate issue. OK, left the code as is.